src/py21cmfast/wrapper.py

```diff
@@ -2643,6 +2646,7 @@ def run_lightcone(
                 hooks=hooks,
                 regenerate=regenerate,
                 direc=direc,
+                write=write,
```

Comment:
please add a comment here indicating this is only a quick hack for running multinest